autogluon/training/docker/0.4/py3/Dockerfile.cpu

```diff
@@ -1,179 +1,47 @@
+    MXNET_CUDNN_AUTOTUNE_DEFAULT=0
 
 # Install dependencies
 RUN apt-get update \
  && apt-get -y upgrade \
  && apt-get install -y --no-install-recommends \
-    gcc \
-    build-essential \
-    wget \
-    curl \
-    emacs \
-    git \
-    vim \
-    unzip \
-    zlib1g-dev \
-    libssl-dev \
     libncurses5-dev \
-    libgdbm-dev \
     libnss3-dev \
     libgl1-mesa-glx \
-    libffi-dev \
     liblzma-dev \
-    libbz2-dev \
-    libsqlite3-dev \
-    tk-dev \
     uuid-dev \
-    ca-certificates \
-    libopencv-dev \
-    openssh-client \
-    openssh-server \
-    libreadline-gplv2-dev \
-    libc6-dev \
-    cmake \
  && apt-get autoremove -y \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-# install OpenSSL
-RUN wget -q -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
- && tar -xzf openssl-${OPENSSL_VERSION}.tar.gz \
- && cd openssl-${OPENSSL_VERSION} \
- && ./config && make -j $(nproc) && make install \
- && ldconfig \
- && cd .. && rm -rf openssl-* \
- && rmdir /usr/local/ssl/certs \
- && ln -s /etc/ssl/certs /usr/local/ssl/certs
-
-# Install Open MPI
-RUN mkdir /tmp/openmpi \
- && cd /tmp/openmpi \
- && wget -q https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.1.tar.gz \
- && tar zxf openmpi-4.0.1.tar.gz \
- && cd openmpi-4.0.1 \
- && ./configure --enable-orterun-prefix-by-default \
- && make -j $(nproc) all \
- && make install \
- && ldconfig \
- && cd /tmp && rm -rf /tmp/openmpi
-
-# Create a wrapper for OpenMPI to allow running as root by default
-RUN mv /usr/local/bin/mpirun /usr/local/bin/mpirun.real \
- && echo '#!/bin/bash' > /usr/local/bin/mpirun \
- && echo 'mpirun.real --allow-run-as-root "$@"' >> /usr/local/bin/mpirun \
- && chmod a+x /usr/local/bin/mpirun \
- && echo "hwloc_base_binding_policy = none" >> /usr/local/etc/openmpi-mca-params.conf \
- && echo "rmaps_base_mapping_policy = slot" >> /usr/local/etc/openmpi-mca-params.conf
-
-ENV LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
-ENV PATH=/usr/local/bin:$PATH
-
-# install Python
-RUN wget -q https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
- && tar -xzf Python-$PYTHON_VERSION.tgz \
- && cd Python-$PYTHON_VERSION \
- && ./configure --enable-shared --prefix=/usr/local \
- && make -j $(nproc) && make install \
- && cd .. && rm -rf ../Python-$PYTHON_VERSION* \
- && ln -s /usr/local/bin/pip3 /usr/bin/pip \
- && ln -s /usr/local/bin/$PYTHON /usr/local/bin/python \
- && pip --no-cache-dir install --upgrade --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
- && pip --no-cache-dir install --upgrade wheel setuptools
-
-WORKDIR /
+ARG AUTOGLUON_VERSION=0.4.2
+ARG TORCH_VER=1.10.2+cpu
+ARG TORCH_VISION_VER=0.11.3+cpu
 
-RUN pip install --no-cache-dir -U \
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U setuptools==59.5.0 wheel \
+ && pip install --no-cache-dir -U \
+    numpy numba \
```

Comment:
Does numpy need an upgrade to latest from the version installed on the mxnet DLC?
https://github.com/aws/deep-learning-containers/blob/e24629a4ac2c6c75880b211cc49d822a2c050b5e/mxnet/training/docker/1.9/py3/Dockerfile.cpu#L98